lib/geminabox/server.rb

```diff
@@ -107,7 +108,9 @@ def dependency_cache
 
     get '/reindex' do
       serialize_update do
-        self.class.reindex(:force_rebuild)
+        params[:force_rebuild] ||= 'true'
+        force_rebuild = params[:force_rebuild] == 'true'
```

Comment:
We should accept only either of `true` or `false`, otherwise

```
error_response(404, "force_rebuild parameter must be either of true or false, but was #{params[:force_rebuild]}")
```